go/exorelay/examples/index.go

```diff
@@ -0,0 +1,23 @@
+package exorelay_examples
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Originate/exocom/go/exorelay"
+	"github.com/Originate/exocom/go/structs"
+)
+
+type Example interface {
+	GetReceivedMessages() []structs.Message
+	Setup(exoRelay *exorelay.ExoRelay)
+}
+
+func Get(name string) Example {
+	switch name {
+	case "message-handler":
+		return &MessageHandlerExample{}
```

Comment:
These are test fixtures used just for feature tests. I'll rename it.

The other two features I had envisioned implementing were [incoming-replies.feature](https://github.com/Originate/exocom/blob/7eec12403b1a18d02037bb680ab00556fbb7b65f/javascript/exorelay/features/incoming-replies.feature) and [outgoing-replies.feature](https://github.com/Originate/exocom/blob/7eec12403b1a18d02037bb680ab00556fbb7b65f/javascript/exorelay/features/outgoing-replies.feature). How do you think those should be dealt with? I'm not certain what to do with the outgoing reply aside from exposing a helper for the user to check if a message is a reply to another